src/components/Mission.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { joinMission } from '../redux/missions/missionsSlice';
+
+const Mission = ({ id, name, description }) => {
+  const dispatch = useDispatch();
+  return (
+    <tr key={id}>
+      <td>{name}</td>
+      <td className="description">{description}</td>
+      <td className="align-middle text-center">Not A Member</td>
+      <td className="align-middle text-center">
+        <Button variant="light" onClick={() => dispatch(joinMission(id))}>Join Mission</Button>
```

Comment:
[Optional] Hello @biruk-bereka, your code works well and the action is dispatched but I think you could implemented the change here.